fastlane/lib/fastlane/actions/xcov.rb

```diff
@@ -35,7 +35,7 @@ def self.available_options
         begin
           Gem::Specification.find_by_name('xcov')
         rescue Gem::LoadError
-          # Catched missing gem exception and returned empty array
+          # Catch missing gem exception and return empty array
```

Comment:
This is not a spelling mistake, right ? I think this needs to be reverted

Reply:
@Kukurijek "catched" → "caught", so it's still a spelling mistake.

That verb could be either in past tense or imperative form, but the "returned" should be "return" instead, as suggested.

I prefer the current spelling as per @jsoref 's suggestion (imperative instead of past tense) for the whole sentence, it reads better :)